lib/accelerometer.js

```diff
@@ -79,8 +79,16 @@ var Controllers = {
     },
     toGravity: {
       value: function(raw) {
+        // Datasheet available at https://www.invensense.com/wp-content/uploads/2015/02/MPU-6000-Datasheet1.pdf
```

Comment:
When I try to visit that url:

![](https://i.gyazo.com/139b8eb2356ec6af99f6aa6a5615c8f8.png)

But when I try it with "http" instead, it works just fine, so let's change the url in the comment to "http"